crates/re_viewer/src/ui/data_ui/image.rs

```diff
@@ -93,15 +96,15 @@ impl DataUi for Tensor {
                         }
                     }
 
-                    if let Some(dynamic_img) = tensor_view.dynamic_img {
+                    if let Some(dynamic_img) = tensor_view.dynamic_img() {
                         // TODO(emilk): support copying and saving images on web
                         #[cfg(not(target_arch = "wasm32"))]
-                        ui.horizontal(|ui| image_options(ui, self, dynamic_img));
+                        ui.horizontal(|ui| image_options(ui, self, &dynamic_img));
 
                         // TODO(emilk): support histograms of non-RGB images too
-                        if let image::DynamicImage::ImageRgb8(rgb_image) = dynamic_img {
+                        if let image::DynamicImage::ImageRgba8(rgb_image) = dynamic_img {
```

Comment:
intentional to switch from rgb to rgba? Won't this break histogram for RGB images (e.g. jpegs)?

Reply:
The DynamicImage is now derived from the cached RGBA ColorImage. I'd like to re-implement the histogram directly on top of the raw tensor so it always represents raw values, but going to save that for a future PR.